fast_calvo_trainer.py

```diff
@@ -0,0 +1,109 @@
+# -----------------------------------------------------------------------------
+# Program Name:         calvo_trainer.py
+# Program Description:  Rodan wrapper for Calvo's classifier training
+# -----------------------------------------------------------------------------
+
+import cv2
+import numpy as np
+import os
+
+from rodan.jobs.base import RodanTask
+import training_engine_sae as training
+
+"""Wrap Patchwise (Fast) Calvo classifier training in Rodan."""
+
+
+class FastCalvoTrainer(RodanTask):
+    name = "Training model for Patchwise Analysis of Music Document"
+    author = "Jorge Calvo-Zaragoza, Francisco J. Castellanos, Gabriel Vigliensoni, and Ichiro Fujinaga"
+    description = "The job performs the training of many Selection Auto-Encoder model for the pixelwise analysis of music document images."
+    enabled = True
+    category = "OMR - Layout analysis"
+    interactive = False
+
+    settings = {
+        'title': 'Training parameters',
+        'type': 'object',
+        'properties': {
+            'Maximum number of training epochs': {
+                'type': 'integer',
+                'minimum': 1,
+                'default': 10
+            },
+            'Patch height': {
+                'type': 'integer',
+                'minimum': 64,
+                'default': 256
+            },
+            'Patch width': {
+                'type': 'integer',
+                'minimum': 64,
+                'default': 256
+            }
+        }
+    }
+
+    input_port_types = (
+        {'name': 'Image', 'minimum': 1, 'maximum': 1, 'resource_types': lambda mime: mime.startswith('image/')},
```

Comment:
We can't handle any type of image MIME types, and so I we should be explicit about TIFF or PNGs

Reply:
Actually, only JPG or PNG can be used, as OpenCV does not read TIFF.